### PR TITLE
YDMU: Priest of Possibility and support

### PR DESCRIPTION
**Pull request by Northmoc**
It's an Alchemy card, but these tweaks might be useful in other situations too... and I don't think they do any harm

### CODE REVIEW

**Comment by Agetian** [comment]
Looks good to me :)

File: forge-game/src/main/java/forge/game/ability/effects/EffectEffect.java [hunk 1]
```diff
@@ -211,10 +199,18 @@ public void resolve(SpellAbility sa) {
...
+                List<String> effectKeywords = Arrays.asList(sa.getParam("Keywords").split(","));
+                if (sa.hasParam("SharedKeywordsZone")) {
+                    List<ZoneType> zones = ZoneType.listValueOf(sa.getParam("SharedKeywordsZone"));
+                    String[] restrictions = sa.hasParam("SharedRestrictions") ? sa.getParam("SharedRestrictions").split(",")
+                            : new String[]{"Card"};
+                    effectKeywords = CardFactoryUtil.sharedKeywords(effectKeywords, restrictions, zones, hostCard, sa);
+                }
+                if (effectKeywords != null) {
+                    for (final String s : effectKeywords) {
+                        eff.addIntrinsicKeyword(s);
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/ability/effects/EffectEffect.java:212]: Having an effect gain keywords feels wrong
Can't you just remember the strings?

Comment (Northmoc) [reply forge-game/src/main/java/forge/game/ability/effects/EffectEffect.java:212]: Sure, could do it that way... the code for putting keywords on Effects was sitting there but unused on any card script, probably just an anachronism

Comment (Northmoc) [reply forge-game/src/main/java/forge/game/ability/effects/EffectEffect.java:212]: I did like the way this version turned out for UI – seemed clean

**Review by tool4ever — CHANGES_REQUESTED** [review]
Should be good after you fixed imports

**Review by tool4ever — APPROVED** [review]

**Pull request merged**